### PR TITLE
manifests: specify system-cluster-critical priority update pull policy

### PR DESCRIPTION
**Pull request by derekwaynecarr**
specifies system-cluster-critical priority for operator.
updates image pull policy to IfNotPresent
blocks passing smoke tests for ensuring control plane pods always schedule.

see: openshift/origin#22217

### CODE REVIEW

File: manifests/06-operator.yaml [hunk 1]
```diff
@@ -31,7 +32,7 @@ spec:
           resources:
             requests:
               cpu: 10m
-          imagePullPolicy: Always
```
Comment (gabemontero) [inline manifests/06-operator.yaml]: PR https://github.com/openshift/cluster-samples-operator/pull/122 just merged with this change

**Comment by gabemontero** [comment]
aws vpc resource failure on e2e-aws-operator run

/test e2e-aws-operator

**Comment by gabemontero** [comment]
@openshift/sig-developer-experience  .. fyi, another hiccup during an image eco run at https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_cluster-samples-operator/123/pull-ci-openshift-cluster-samples-operator-master-e2e-aws-image-ecosystem/314 ... this time on image push:

```
2019-03-04T20:27:58.669312017Z error: build error: 1 error occurred:
2019-03-04T20:27:58.669312017Z 	* Error writing blob: error storing blob to image destination for cache "containers-storage:[overlay@/var/lib/containers/storage+/var/run/containers/storage:overlay.skip_mount_home=false]image-registry.openshift-image-registry.svc:5000/openshift/python@sha256:78952d9e0acba0a5f95a6a2a8a899cab0bef3a687b2f63c5a77ef17946144b60": error storing blob to file "/var/tmp/storage273933876/5": unexpected EOF
2019-03-04T20:27:58.669312017Z
```

I think I recall from the last hiccup on the pull image side that there was a statement that there was retry already in play for push.

Should this have seen a retry?

First instance of this I've seen with image eco.

**Comment by bparees** [comment]
> Should this have seen a retry?

i think it's a question for @nalind.  Generally yes our pushes are wrapped in a retry logic, i'm not sure if this particular error is handled differently. It's also a generally worrying error (it looks like a failure on the client side filesystem, not the remote registry).

**Comment by gabemontero** [comment]
@derekwaynecarr would you like for me to pick your commit into a PR of my own so I can handle the file conflict and babysit the e2e tests to closure ?

**Comment by gabemontero** [comment]
using https://github.com/openshift/cluster-samples-operator/pull/126 now

**Pull request closed without merging**